packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6865,10 +6847,14 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return this.mediaProperties.webrtcMediaConnection
       .update({
+        // TODO: RoapMediaConnection is not ready to use stream classes yet, so we pass the raw MediaStreamTrack for now
         localTracks: {
-          audio: this.mediaProperties.audioTrack?.underlyingTrack || null,
-          video: this.mediaProperties.videoTrack?.underlyingTrack || null,
-          screenShareVideo: this.mediaProperties.shareVideoTrack?.underlyingTrack || null,
+          audio: this.mediaProperties.audioStream?.outputStream.getTracks()[0] || null,
```

Comment:
I think it would be better to use `outputTrack` instead of `outputStream.getTracks()[0]`

Reply:
Done: https://github.com/webex/webex-js-sdk/pull/3052